autolinks.bash

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Adevinta
+
+# substitute_autolinks creates autolinks in the provided file if the
+# autolink ends with the provided delimiter.
+substitute_autolinks() {
+	local file=$1
+	local delim=$2
+
+	sed -i -E 's/(^|^[^\t].*?[[:space:]])(https?:\/\/[[:alnum:][:punct:]]+?)('"${delim}"')/\1<\2>\3/gm' "${file}"
```

Comment:
This doesn't works on mac.

- gsed should be installed/used instead.
- Also the dependencies are installed for linux.

I understand this is not a requirement so no big deal.

Reply:
1060083c9fd5e6c66398047ca3c3c42ccb5e2b8a moves build and test to docker. PTAL.